tests/suites/test_suite_mpi.data

```diff
@@ -292,6 +594,63 @@ mbedtls_mpi_cmp_mpi:"-1230000000000000000":"":-1
 Test mbedtls_mpi_cmp_mpi: large negative < 0 (1 limb)
 mbedtls_mpi_cmp_mpi:"-1230000000000000000":"0":-1
 
+Base test mbedtls_mpi_core_lt_ct #1
+mbedtls_mpi_core_lt_ct:"02B5":"02B5":0
+
+Base test mbedtls_mpi_core_lt_ct #2
+mbedtls_mpi_core_lt_ct:"02B5":"02B4":0
+
+Base test mbedtls_mpi_core_lt_ct #3
+mbedtls_mpi_core_lt_ct:"02B5":"02B6":1
+
+Base test mbedtls_mpi_core_lt_ct (length=0)
+mbedtls_mpi_core_lt_ct:"":"":0
+
+Base test mbedtls_mpi_core_lt_ct (corner case - 64 bit) #1
```

Comment:
In new test case descriptions, please write meaningful descriptions. Only use `#1`, `#2`, etc. if there's absolutely no way to describe how those test cases differ, typically because the test cases are identical except for using different pseudorandom inputs.

We have a display limit of 66 characters. Please avoid prefixes that don't convey any meaning such as “Test” or “Base test” unless expanding a sequence that already has this format. (I never understood what makes some test cases “base” anyway.) See for example the `mbedtls_mpi_swap` test cases in `test_suite_mpi.data` for what I consider to be a nice format for MPI test case descriptions.